server/routes/support/handlers/probationTeam.ts

```diff
@@ -21,16 +22,22 @@ export default class ProbationTeamRoutes {
       view === 'prison'
         ? await this.comCaseloadService.getTeamCreateCaseload(user, [teamCode])
         : await this.comCaseloadService.getTeamVaryCaseload(user, [teamCode])
-    ).map(comCase => {
-      const releaseDate = comCase.releaseDate ? format(parseCvlDate(comCase.releaseDate), 'dd MMM yyyy') : 'not found'
-      return {
-        ...comCase,
-        releaseDate,
-        hardStopDate: comCase.hardStopDate && format(parseCvlDate(comCase.hardStopDate), 'dd/MM/yyyy'),
-        hardStopWarningDate:
-          comCase.hardStopWarningDate && format(parseCvlDate(comCase.hardStopWarningDate), 'dd/MM/yyyy'),
-      }
-    })
+    )
+      .map(comCase => {
+        const releaseDate = comCase.releaseDate ? format(parseCvlDate(comCase.releaseDate), 'dd MMM yyyy') : 'not found'
+        return {
+          ...comCase,
+          releaseDate,
+          licenceStatus: comCase.isReviewNeeded ? LicenceStatus.REVIEW_NEEDED : comCase.licenceStatus,
```

Comment:
Is this not also needed for the `probationStaff` vary caseload?

Reply:
yes, it should be, added